### PR TITLE
Update item_charge_bkb.txt

### PR DESCRIPTION
**Pull request by RamonNZ**
Change the current item_Charge_BKB with this, which is fully working utilising the Charge_BKB.lua script. 

It includes my explanations and rationales for how KVs are utilized.

### CODE REVIEW

**Comment by warpdragon** [comment]
cool

**Comment by RamonNZ** [comment]
I may need to update it. I think I broke something playing with it today.

File: game/scripts/npc/items/item_charge_bkb.txt [hunk 1]
```diff
@@ -49,42 +50,138 @@
...
+				"immunity_time_per_charge"	"2"		//RamonNZ: New KV - How many seconds of spell immunity you get per charge when activated.
 			}
 		}
-	}
\ No newline at end of file
+		"OnSpellStart"
+		{
+			"RunScript"
+			{
+				"ScriptFile"			"items/charge_bkb.lua"
```
Comment (chrisinajar) [inline game/scripts/npc/items/item_charge_bkb.txt]: You're missing this file, you only have `game/scripts/npc/items/item_charge_bkb.txt` in this PR

Comment (RamonNZ) [inline game/scripts/npc/items/item_charge_bkb.txt]: That script is in another PR

**Pull request closed without merging**